packages/edit-post/src/components/layout/style.scss

```diff
@@ -124,6 +124,11 @@
 		margin-left: auto;
 		margin-right: auto;
 
+		// Provide every block with a default base margin. This margin provides a consistent spacing
+		// between blocks in the editor.
+		margin-top: $default-block-margin;
+		margin-bottom: $default-block-margin;
```

Comment:
This is now has a bit more specificity than the previous block margins for classic themes. This is probably the most impactful thing but I don't think we have another way to apply these to only the classic themes without the added specificity. (There might be another way by adding an inline style in php but I'd rather not do it unless necessary :P)